cmd/notification.go

```diff
@@ -1079,6 +1079,29 @@ func (sys *NotificationSys) CPUInfo() []madmin.ServerCPUHardwareInfo {
 	return reply
 }
 
+// NETWORKInfo - Network Hardware info
+func (sys *NotificationSys) NETWORKInfo() []madmin.ServerNetworkHardwareInfo {
+	reply := make([]madmin.ServerNetworkHardwareInfo, len(sys.peerClients))
+	var wg sync.WaitGroup
+	for i, client := range sys.peerClients {
+		if client == nil {
+			continue
+		}
+		wg.Add(1)
+		go func(client *peerRESTClient, idx int) {
+			defer wg.Done()
+			network, err := client.NETWORKInfo()
```

Comment:
```suggestion
			netinfo, err := client.NetworkInfo()
```

Is the correct naming convention and variable name usage

Reply:
thanks . changed.